hawkular-bus-common/src/main/java/org/hawkular/bus/common/MessageProcessor.java

```diff
@@ -487,26 +456,14 @@ protected Message createMessageWithBinaryData(ConnectionContext context, BasicMe
         // we are using a ActiveMQ-specific feature that allows us to stream blobs
```

Comment:
this three-line comment can go away - looks like the new session API provides what ActiveMQ did not.